OAuthSwift.xcodeproj/xcshareddata/xcschemes/OauthSwiftDemo.xcscheme

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I didn't check if it's a new file or rename but other files use `OAuth*` name yet this file has `Oauth`.